docs/Deploying/Server-Installation.md

```diff
@@ -447,6 +447,11 @@ spec:
       - name: fleet
         image: fleetdm/fleet:4.18.0
         env:
+          # if running Fleet behind external ingress controller that terminates TLS
+          - name: FLEET_SERVE_TLS
```

Comment:
Should this be FLEET_SERVER_TLS?

Reply:
@edwardsb Can you please confirm whether this should be FLEET_SERVER_TLS?

Reply:
It should be. Thanks @RachelElysia